gcm-sender-async/src/test/java/org/whispersystems/gcm/server/SenderTest.java

```diff
@@ -141,7 +141,7 @@ void testServerErrorRecovery() throws InterruptedException, ExecutionException,
     CompletableFuture<Result> future = sender.send(Message.newBuilder().withDestination("1").build());
 
     // up to three failures can happen, with 100ms exponential backoff
-    // if we end up using the fourth, and finaly try, it would be after ~700 ms
+    // if we end up using the fourth, and finally try, it would be after ~700 ms
```

Comment:
```suggestion
    // if we end up using the fourth, and final try, it would be after ~700 ms
```
Ultra nitpick, but I believe the `y` is the typo, not the missing `l`